### DISCUSSION TITLE
在2.11.3版本中菜单切换时页面空白

### DISCUSSION THREAD

**Discussion by ShuangxiHan** [vbenjs/vue-vben-admin#3789] [Q&A]
![image](https://github.com/vbenjs/vue-vben-admin/assets/143981998/a4551e4c-cf02-4bed-b5aa-35a4ede07598)
在菜单切换时，点击个别菜单会出现页面没有渲染，但是刷新一下页面就可以了，而且这种问题菜单是不固定的，但是那个合作伙伴的菜单是固定出问题的，不知道是什么原因

**Comment by warmthsea** [comment]
可以提供具体的细节么？
比如那个菜单那个路由
合作伙伴的菜单我没有找到
